src/apsbits/core/run_engine_init.py

```diff
@@ -9,14 +9,14 @@
 import logging
 
 import bluesky
+from apstools.utils import StoredDict
```

Comment:
Why this does not work here @prjemian ?

Reply:
Not in latest apstools release 1.7.2.

Reply:
Even when apstools is installed from a pre-release, need to check that all capability here is available from apstools before this directory is removed.